test/system/410-selinux.bats

```diff
@@ -50,6 +50,14 @@ function check_label() {
     check_label "--systemd=always" "container_init_t"
 }
 
+@test "podman selinux: init container with --security-opt type" {
+    check_label "--systemd=always --security-opt=label=type:spc_t" "spc_t"
+}
+
+@test "podman selinux: init container with --security-opt level" {
+    check_label "--systemd=always --security-opt=label=level:s0:c1,c2" "container_init_t"
```

Comment:
This test passes, but it also passes in podman without your PR. (IOW, this is a test of functionality already present in podman. Which is fine by me, I like extra tests, but I wanted to make sure you weren't expecting this to be a test for your fix).

Reply:
I want to make sure my changes do not change this behavior. IE I want to make sure we still get container_init_t if we change the level without changing the type.